packages/components/ContextualMenu/src/ContextualMenuItem.settings.ts

```diff
@@ -11,6 +11,7 @@ export const settings: IComposeSettings<ContextualMenuItemType> = [
     root: {
```

Comment:
should there be an empty default style for the container, like content/icon (in expanded diff)?

Reply:
nm I misread, this is ContextualMenuItem.settings :)